Views/Home/Specs.cshtml

```diff
@@ -0,0 +1,13 @@
+@{

```

Comment:
an example of a view that gets data passed from a form and displays some results